### ISSUE TITLE
A11y_AzureExtensionsforVSCode_AzureFuction_CreateNewProject__ColorContrast : The color contrast ratio of keyboard focus on several controls such as "Python", "JavaScript", etc. is less than 3:1

### ISSUE THREAD

**Issue by nirmalsahoo** [microsoft/vscode#104556]
GitHub Tags: #A11y_VSCode;#CAITestPass_May2020;#Visual Studio Code;#A11yTCS;#A11yWCAG2.1;#MAC;#FTP;#Benchmark;#ColorContrast;#A11ySev2:#MAS1.4.11;

**Environment Details:**
Application Name: Azure Extensions for VS Code
Application Version: 1.48.0-insider
Commit: cfbd1999769f4f08dce29629fb92fdc0fac53829
Date: 2020-08-06T05:40:40.871Z
Electron: 7.3.2
Chrome: 78.0.3904.130
Node.js: 12.8.1
V8: 7.8.279.23-electron.0
OS: Darwin x64 19.6.0
Operating system: macOS Catalina Version 10.15.6 (19G73)
Screen Reader: VoiceOver
MAS References: MAS1.4.11

**Repro Steps:**
Open Visual studio code
Click on extension button and search for "Azure Functions"
Click on install and move to the "Azure" button
Click on "Create New Project Button" from the toolbar
Navigate to options from the list box such as "Python", "JavaScript", etc. control using keyboard and verify if the color contrast ratio of keyboard focus is less than 3:1 or not.

**Actual:**
The several controls such as "Python", "JavaScript", etc. has keyboard focus color contrast ratio of 1.1:1 which is less than the standard color contrast ratio of 3:1 which is not appropriate as the users with low vision will have difficulty in understanding the navigation on the screen.

**Expected:**
The keyboard focus color contrast ratio of several controls such as "Python", "JavaScript", etc. should be greater than or equal to 3:1.

Attachment link for Reference:

**Comment by misolori** [comment]
@nirmalsahoo I don't believe you grabbed the correct colors. `#F3F3F3` is the background color of the quick pick, the text foreground is `#616161` and this should pass:

![image](https://user-images.githubusercontent.com/35271042/90256141-1f9a7480-ddfa-11ea-8732-a6e1af5191e9.png)
